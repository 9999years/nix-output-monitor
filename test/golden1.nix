{ seed, system ? builtins.currentSystem }:
let
  mkBuild = name: deps: derivation {
    inherit name deps seed system;
    builder = ./golden1-builder.sh;
  };
in
rec {
  build1 = mkBuild "build1" [ ];
  build2 = mkBuild "build2" [ build1 ];
  build3 = mkBuild "build3" [ build1 build2 ];
  build4 = mkBuild "build4" [ build2 build3 ];
}
