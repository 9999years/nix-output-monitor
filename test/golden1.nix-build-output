these 2 derivations will be built:
  /nix/store/38p9jnjnh8da33nqyhwviih5rgfbfkg6-remote-build2.drv
  /nix/store/3dhczfasgzbch0wl0jlqs4qivghxcjz5-local-build.drv
building '/nix/store/38p9jnjnh8da33nqyhwviih5rgfbfkg6-remote-build2.drv' on 'ssh://maralorn@phoibe.cased.de'...
copying 0 paths...
copying 1 paths...
copying path '/nix/store/3k988pzwk26rxx9l4pga1cqcddz314bd-remote-build2' from 'ssh://maralorn@phoibe.cased.de'...
building '/nix/store/3dhczfasgzbch0wl0jlqs4qivghxcjz5-local-build.drv'...
