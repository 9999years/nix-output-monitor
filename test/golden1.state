BuildState {outstandingBuilds = fromList [], outstandingDownloads = fromList [], plannedCopies = 2, runningBuilds = fromList [(Localhost,fromList []),(Host "ssh://example-host.com",fromList [])], completedBuilds = fromList [(Localhost,fromList [Derivation {toStorePath = StorePath {hash = "25fhmgnm1rg6aqvr20n7l02f15ygvzxn", name = "local-build"}}]),(Host "ssh://example-host.com",fromList [Derivation {toStorePath = StorePath {hash = "f8skr8yq76q3z1lm8z8i2l6y046c0axx", name = "remote-build"}}])], failedBuilds = fromList [], completedDownloads = fromList [(Host "ssh://example-host.com",fromList [StorePath {hash = "5gki8h3p9g63mp9q6xy9z7hlq1haqq6j", name = "remote-build"}])], completedUploads = fromList [(Host "ssh://example-host.com",fromList [StorePath {hash = "gghmji2b9icmh36f4f9swz1s2jkrppri", name = "golden1-builder.sh"}])], outputToDerivation = fromList [(StorePath {hash = "5gki8h3p9g63mp9q6xy9z7hlq1haqq6j", name = "remote-build"},Derivation {toStorePath = StorePath {hash = "f8skr8yq76q3z1lm8z8i2l6y046c0axx", name = "remote-build"}}),(StorePath {hash = "sp8ra6al9vygv37grlkfc8625lljhyxa", name = "local-build"},Derivation {toStorePath = StorePath {hash = "25fhmgnm1rg6aqvr20n7l02f15ygvzxn", name = "local-build"}})], derivationToOutput = fromList [(Derivation {toStorePath = StorePath {hash = "25fhmgnm1rg6aqvr20n7l02f15ygvzxn", name = "local-build"}},StorePath {hash = "sp8ra6al9vygv37grlkfc8625lljhyxa", name = "local-build"}),(Derivation {toStorePath = StorePath {hash = "f8skr8yq76q3z1lm8z8i2l6y046c0axx", name = "remote-build"}},StorePath {hash = "5gki8h3p9g63mp9q6xy9z7hlq1haqq6j", name = "remote-build"})], lastPlannedBuild = Just (Derivation {toStorePath = StorePath {hash = "f8skr8yq76q3z1lm8z8i2l6y046c0axx", name = "remote-build"}}), buildReports = fromList [], startTime = 2021-03-04 00:35:28.627866489 UTC, errors = []}
