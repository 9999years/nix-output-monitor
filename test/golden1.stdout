/nix/store/z41m8147d2zrkp1rfxrsw8r0mylzzqc8-build1
/nix/store/x8r7rjimhn6alyln68hbryqhjbapkzq1-build2
/nix/store/b4601pj97cjf6z6vrwyisrychh0n0007-build3
/nix/store/pq5i0crbvdp8rljvibgc87bzw3xbf88d-build4
